0008-no-caret-prerelease-installs.md

```diff
@@ -0,0 +1,47 @@
+# No Caret Ranges for Prerelease Installs
+
+## Summary
+
+This proposal changes the default behavior of the install command to pin the package version, but only for a package that is tagged with a version that is a [prerelease](https://github.com/npm/node-semver#prerelease-tags) (and not add the [caret range](https://github.com/npm/node-semver#caret-ranges-123-025-004), `^`)
+
+```sh
+# if this resolves to a prerelease such as 7.0.0-beta.52
+npm install @babel/core
+```
+
+results in
+
+```diff
+"devDependencies": {
+-   "@babel/register": "^7.0.0-beta.52"
++   "@babel/register": "7.0.0-beta.52"
```

Comment:
a bit odd that it says `@babel/register` here when it's `@babel/core` in the install command above 😅

Reply:
lol good point, feel free to PR or I can later

Reply:
Can do, simple enough. Just wasn't sure if it was worth a PR